src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -399,6 +412,32 @@ private void FinishSolutionBindingOnUIThread(IProgressController controller, Can
 
         #region Helpers
 
+        private async Task<T> SafeServiceCall<T>(Func<Task<T>> call, IProgressController controller, CancellationToken token)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter 'controller'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter 'token'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1172)